docs/api/app.md

```diff
@@ -965,7 +968,8 @@ starts:
 const { app } = require('electron')
 let myWindow = null
 
-const gotTheLock = app.requestSingleInstanceLock()
+const additionalData = { myKey: 'myValue' }
+const gotTheLock = app.requestSingleInstanceLock(additionalData)
```

Comment:
This example should be updated to show usage of `additionalData` in responding to the `second-instance` event